lib/terraform/runner/api_params.rb

```diff
@@ -42,6 +43,74 @@ def self.to_cam_parameters(vars)
           to_cam_param(key, value)
         end
       end
+
+      require 'set'
+      TRUE_VALUES = ['T', 't', true, 'true', 'True', 'TRUE'].to_set
```

Comment:
You were using `ActiveModel::Type::Boolean.new.cast` above any reason to not continue to use that here?

Reply:
Yes, there is reason, we are using `TRUE_VALUES`.

And `ActiveModel::Type::Boolean`, uses `FALSE_VALUES`, https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html .

It is more to do with ManageIQ Checkbox field,  though we configure it with boolean `true` or `false` when the field is selected, the result is string `'t'`, whereas when unchecked it blank string `''`.

Terraform considers blank string `''` as invalid for boolean type.
 So we can convert `''` to `nil`/`null`, which is expectable  for Terraform.

In fact `ActiveModel::Type::Boolean` converts `''` to  `nil`. 

But `''` or `nil`/`null` cannot be always same as `false`.